exercises/yacht/canonical-data.json

```diff
@@ -0,0 +1,261 @@
+{
+    "exercise" : "yacht",
+    "version" : "1.0.0",
+    "comments" : [
+        "The dice are represented always as a list of exactly five integers",
+        "with values beteen 1 and 6 inclusive. The category is an integer.",
+        "the numbers 0 to 5 represent the categories 'ones' to 'sixes'",
+        "Then 6 is full house,",
+        "     7 is four of a kind",
+        "     8 is a little straight 1-5",
+        "     9 is b big straight 2-6",
```

Comment:
I don't think `b big straight` is correct here. I think it should be `a bit straight`, unless the `b` has a special significance I missed.

Reply:
EBWODP:

> a bit straight

I was wrong as well.

It should be `a big straight`